spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/TaskExecutionInfoService.java

```diff
@@ -28,11 +28,19 @@
  * @author Daniel Serleg
  * @author Mark Pollack
  * @author David Turanski
+ * @author Glenn Renfro
  */
 public interface TaskExecutionInfoService {
 
+	/**
+	 * Create a the {@link TaskExecutionInformation} instance for the information provided.
+	 * @param taskName the name of the task definition
+	 * @param taskDeploymentProperties the deployment properties to use for the {@link TaskExecutionInformation}
+	 * @param userComposedTaskRunnerName user provided CTR app name to use.
+	 * @return instance of {@link TaskExecutionInformation}
+	 */
 	TaskExecutionInformation findTaskExecutionInformation(String taskName,
-			Map<String, String> taskDeploymentProperties);
+			Map<String, String> taskDeploymentProperties, String userComposedTaskRunnerName);
```

Comment:
'user' feels a bit odd as a prefix for the argumetn, maybe 'composedTaskRunnerName' or 'customComposedTaskRunnerName'

Reply:
makes sense.